commands/cli.go

```diff
@@ -115,7 +115,8 @@ func awsNuke(c *cli.Context) error {
 			}
 		}
 	} else {
-		logging.Logger.Infoln("The --force flag is set, so proceeding without confirmation.")
+		logging.Logger.Infoln("The --force flag is set, so proceeding after 10 seconds without confirmation.")
```

Comment:
I'd update this to say: "The --force flag is set, so waiting for 10 seconds before proceeding to nuke everything your account. If you don't want to proceed, hit CTRL+C now!!"